### DISCUSSION TITLE
ext clock for the ACQ430FMC

### DISCUSSION THREAD

**Discussion by nilsbuc** [PandABlocks/PandABlocks.github.io#12] [Q&A]
Hey,
Is the ACQ430FMC connected to the internal clock of the PandAbox, or does it depend on which clock I'm using. So if I'm connecting an external clock  to the PandAbox and switch to this via the system control block, will the ACQ430FMC be triggered by the external clock?

Thanks in advance 
Nils

**Comment by coretl** [comment]
The ACQ430 will always be triggered with the internal clock as fast as it can go, the external clock input is exposed just as another input on the block. Our use-case takes the external trigger signal and uses it with the PCAP block to average ADC values for given time slices, so the clock largely doesn't matter for us. What is your use case?
